src/app/app.component.scss

```diff
@@ -0,0 +1,7 @@
+h1,a {
+    margin: 10px 10px 10px 10px;
```

Comment:
You can use bootstrap margin classes instead. For example m-1, m-2 etc.
Also you could have just written: margin: 10px;

Reply:
![image](https://user-images.githubusercontent.com/45048289/75234785-7b753080-57bb-11ea-94a8-593924bd7809.png)

I've done it. I am wondering - Can I use it in this way and set style for a m-1 class? Because for now I've done it using separate class from mdb for h1 and p tags (font-weight-bold and text-warning)

Reply:
I already know - I could use for example :first-child

Reply:
I don't know what you were talking about exactly.